SPECS/kernel/kernel-uki.spec

```diff
@@ -65,11 +65,19 @@ install -vdm 700 %{buildroot}/lib/modules/%{kernelver}
 install -vm 600 vmlinuz-uki.efi %{buildroot}/boot/vmlinuz-uki-%{kernelver}.efi
 ln -s /boot/vmlinuz-uki-%{kernelver}.efi %{buildroot}/lib/modules/%{kernelver}/vmlinuz-uki.efi
 
+# install to ESP
+install -vdm 700 %{buildroot}/boot/efi/EFI/Linux
+cp %{buildroot}/boot/vmlinuz-uki-%{kernelver}.efi %{buildroot}/boot/efi/EFI/Linux/vmlinuz-uki-%{kernelver}.efi
+
 %files
 /boot/vmlinuz-uki-%{kernelver}.efi
 /lib/modules/%{kernelver}/vmlinuz-uki.efi
+/boot/efi/EFI/Linux/vmlinuz-uki-%{kernelver}.efi
```

Comment:
Is there a particular reason why we chose "EFI/Linux" for the directory name? Is this naming choice part of a standard?

Reply:
That's the preferred location to store UKIs according to the [Boot loader specification](https://uapi-group.org/specifications/specs/boot_loader_specification/#type-2-efi-unified-kernel-images). systemd-boot follows this spec to search for UKIs under $BOOT/EFI/Linux